basic_example/graph_client_basic_example.go

```diff
@@ -22,16 +23,16 @@ const (
 )
 
 // Initialize logger
-var log = nebula.DefaultLogger{}
+var log = nebulago.DefaultLogger{}
```

Comment:
why change the package name?

Reply:
Distinguish `github.com/vesoft-inc/nebula-go/v2` and `github.com/vesoft-inc/nebula-go/v2/nebula`

Reply:
`nebulaCli` seems to be a more appropriate name here.